### ISSUE TITLE
Missing external function on Windows

### ISSUE THREAD

**Issue by mulimoen** [kornelski/openjpeg-sys#3]
On building on the default github runner `windows-latest`, a linking error occurs:
```console
error LNK2019: unresolved external symbol posix_memalign referenced in function opj_aligned_alloc_n
```
It seems `OPJ_HAVE_POSIX_MEMALIGN` is unconditionally defined, despite not being available on the current platform.

**Comment by kornelski** [comment]
Thanks. Fixed in 769919071ce9667bed346b277a957c72e9570dd7

**Comment by mulimoen** [comment]
Thanks, this fixed the problem for me.
